source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -803,6 +803,59 @@ namespace RendererQml
 
         return uiDateInput;
     }
+	
+	std::shared_ptr<QmlTag> AdaptiveCardQmlRenderer::FactSetRender(std::shared_ptr<AdaptiveCards::FactSet> factSet, std::shared_ptr<AdaptiveRenderContext> context)
+	{
+		auto uiFactSet = std::make_shared<QmlTag>("Column");
+
+		if (!factSet->GetIsVisible())
+		{
+			uiFactSet->Property("visible", "false");
+		}
+
+		for (const auto fact : factSet->GetFacts())
+		{
+			auto uiRow = std::make_shared<QmlTag>("RowLayout");
+
+			auto factTitle = std::make_shared<AdaptiveCards::TextBlock>();
+
+			factTitle->SetText(fact->GetTitle());
+			factTitle->SetTextSize(context->GetConfig()->GetFactSet().title.size);
+			factTitle->SetTextColor(context->GetConfig()->GetFactSet().title.color);
+			factTitle->SetTextWeight(context->GetConfig()->GetFactSet().title.weight);
+			factTitle->SetIsSubtle(context->GetConfig()->GetFactSet().title.isSubtle);
+			factTitle->SetWrap(context->GetConfig()->GetFactSet().title.wrap);
+
+			//TODO: cpp Object Model does not support max width.
+			//factTitle->SetMaxWidth(context->GetConfig()->GetFactSet().title.maxWidth);
 
+			auto uiTitle = context->Render(factTitle);
+
+			//uiTitle->Property("spacing", std::to_string(context->GetConfig()->GetFactSet().spacing));
+			
+			auto factValue = std::make_shared<AdaptiveCards::TextBlock>();
+
+			factValue->SetText(fact->GetValue());
+			factValue->SetTextSize(context->GetConfig()->GetFactSet().value.size);
+			factValue->SetTextColor(context->GetConfig()->GetFactSet().value.color);
+			factValue->SetTextWeight(context->GetConfig()->GetFactSet().value.weight);
+			factValue->SetIsSubtle(context->GetConfig()->GetFactSet().value.isSubtle);
+			factValue->SetWrap(context->GetConfig()->GetFactSet().value.wrap);
+			// MaxWidth is not supported on the Value of FactSet. Do not set it.
+
+			auto uiValue = context->Render(factValue);
+
+			if (factValue->GetWrap())
+			{
+				uiValue->Property("wrapMode", "Text.Wrap");
+			}
```

Comment:
Do we need this here... this is already managed in the `TextBlockRender`.

Reply:
Yeah, will modify it